reading/atoms.livemd

```diff
@@ -115,18 +115,18 @@ Under the hood, `nil` is an atom `:nil`.
 We can use `===` to verify that `:nil` and `nil` are equivalent.
 
 ```elixir
-nil === :nil
```

Comment:
This should stay `:nil` unfortunately, the formatter automatically replaces `:nil` with `nil`.